boards/FSM/Src/App/App_Brake.c

```diff
@@ -0,0 +1,70 @@
+#include <stdlib.h>
+#include <assert.h>
+#include "App_SharedBinaryStatus.h"
+#include "App_InRangeCheck.h"
+
+struct Brake
+{
+    struct InRangeCheck *brake_pressure_in_range_check;
+    struct BinaryStatus *is_brake_open_or_short_circuit_binary_status,
+        *is_brake_actuated_binary_status;
+
+    float min_brake_pressure_psi;
+    float max_brake_pressure_psi;
+
+    float (*get_pressure_psi)(void);
+    bool (*is_open_or_short_circuit)(void);
```

Comment:
bit of a nit, but I think `pressure_sensor_has_open_short_circuit` just to be _super_ clear.

Reply:
Yeah I think that would be nice, since the concept of brake doesn't short circuit. It's the underlying sensor that does.

On a semi-related note, I'd probably rename all instances of `brake_pressure` to `pressure`  (or the other way around) for consistency